# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eterm/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This patch was taken from Debian Linux.
Author: Andreas Jochens
Bug#297917: eterm: FTBFS (amd64/gcc-4.0): invalid lvalue in assignment 
URL: http://www.mail-archive.com/debian-bugs-rc@lists.debian.org/msg17184.html

--- Eterm-0.9.3/src/command.c.orig	2006-04-04 18:11:41.000000000 +0200
+++ Eterm-0.9.3/src/command.c	2006-04-04 18:07:49.000000000 +0200
@@ -3002,9 +3002,9 @@
     Xfd = XConnectionNumber(Xdisplay);
     D_CMD(("Xfd = %d\n", Xfd));
     cmdbuf_ptr = cmdbuf_endp = cmdbuf_base;
-    AT_LEAST((int) num_fds, Xfd + 1);
+    AT_LEAST(num_fds, Xfd + 1);
     if (pipe_fd >= 0) {
-        AT_LEAST((int) num_fds, pipe_fd + 1);
+        AT_LEAST(num_fds, pipe_fd + 1);
     }
     if ((cmd_fd = command_func(argv)) < 0) {
         print_error("Unable to run sub-command.\n");
